playbooks/test-redis.yml

```diff
@@ -2,7 +2,7 @@
 - hosts: ["localhost"]
   become: true
   roles:
-    - role: "famedly.base.redis"
+    - role: "redis"
```

Comment:
ooi: why?

Reply:
it's also in the commit description, but: The Github environment is being weird and doesn't find the role otherwise, despite the same setup working locally